code/game/area/misc_areas.dm

```diff
@@ -35,6 +35,9 @@
 /area/space/firereset(obj/source)
 	return
 
+/area/space/centcomm
+	icon_state = "space_cc"
```

Comment:
Please edit `/datum/map_per_tile_test/structures_in_farspace_checker` so that it still includes these, but doesn't include `/area/space/nearstation`

Reply:
Done!